runecoral/runecoral.h

```diff
@@ -28,6 +28,8 @@ typedef enum {
 typedef struct {
   // What type of elements does this tensor contain?
   RuneCoralElementType type;
+  // C style string to the tensor's name
+  const char *name;
```

Comment:
Is it worth mentioning who owns this string?

Reply:
Nah.. this is just like the shape field. Can come from rust side too (currently, wont because we didnt add the api to Tensor/TensorMut)